### PR TITLE
Consul UI Screenshot Updates

### PR DESCRIPTION
**Pull request by jherschman**
Updated outdated screenshots throughout Consul Docs site.

### CODE REVIEW

File: website/content/docs/connect/observability/ui-visualization.mdx [hunk 1]
```diff
@@ -27,7 +27,7 @@ URLs](#configuring-dashboard-urls).
 It is possible to configure the UI to fetch basic metrics from your metrics
 provider storage to augment the visualization as displayed below.
 
-![Consul UI Service Mesh Visualization](/img/ui-service-topology-view.png)
+![Consul UI Service Mesh Visualization](/img/ui-service-topology-view-hover.png)
```
Comment (karl-cardenas-coding) [inline website/content/docs/connect/observability/ui-visualization.mdx:30]: This image is not rendering. I don't see this image added to the `website/public/img/` directory.

File: website/content/docs/k8s/connect/observability/metrics.mdx [hunk 2]
```diff
@@ -112,7 +112,7 @@ Consul’s built-in UI has a topology visualization for services part of the Con
 
 The diagram below illustrates how the UI displays service metrics for a sample application:
 
-[![UI Topology View](/img/ui-topology.png)](/img/ui-topology.png)
+[![UI Topology View](/img/ui-service-topology-view-hover.png)](/img/ui-service-topology-view-hover.png)
```
Comment (karl-cardenas-coding) [inline website/content/docs/k8s/connect/observability/metrics.mdx:115]: Is there a reason for why this being wrapped inside a `[]` as a whole, then followed by the parentheses (path) ?

File: website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx [hunk 3]
```diff
@@ -441,7 +441,7 @@ to retrieve the bootstrap token mentioned in the UI documentation.
 With the UI open, you'll be able to switch between datacenters via the dropdown
 in the top left:
 
-![Consul Datacenter Dropdown](/img/consul-datacenter-dropdown.png 'Consul Datacenter Dropdown')
+![Consul Datacenter Dropdown](/img/data-center-dropdown.png 'Consul Datacenter Dropdown')
```
Comment (karl-cardenas-coding) [inline website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx:444]: This image is not rendering.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Two of the images are not rendering.

File: website/content/docs/connect/observability/ui-visualization.mdx (hunk 1 above)
Comment (jherschman) [reply website/content/docs/connect/observability/ui-visualization.mdx:30]: Yup my bad, I forgot to commit that folder! Updating now

File: website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx (hunk 3 above)
Comment (jherschman) [reply website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx:444]: images should be there now!

File: website/content/docs/k8s/connect/observability/metrics.mdx (hunk 2 above)
Comment (jherschman) [reply website/content/docs/k8s/connect/observability/metrics.mdx]: good question, I just left it how i found it and didn't realize, it has now been updated.

**Review by karl-cardenas-coding — APPROVED** [review]
Awesome images 👍🏼

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/546794.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 66c5c8f2b537b30e2f760b24e2066bfab6ccc1c6 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 66c5c8f2b537b30e2f760b24e2066bfab6ccc1c6 onto `release/1.11.x` succeeded!

**Pull request merged**